src/renderer/features/player/components/playerbar.tsx

```diff
@@ -70,14 +72,21 @@ export const Playerbar = () => {
     const player = useCurrentPlayer();
     const muted = useMuted();
     const { autoNext } = usePlayerControls();
+    const { expanded: isFullScreenPlayerExpanded } = useFullScreenPlayerStore();
+    const setFullScreenPlayerStore = useSetFullScreenPlayerStore();
+
+    const handleToggleFullScreenPlayer = (e?: MouseEvent<HTMLDivElement> | KeyboardEvent) => {
+        e?.stopPropagation();
+        setFullScreenPlayerStore({ expanded: !isFullScreenPlayerExpanded });
+    };
 
     const autoNextFn = useCallback(() => {
         const playerData = autoNext();
         updateSong(playerData.current.song);
     }, [autoNext]);
 
     return (
-        <PlayerbarContainer>
+        <PlayerbarContainer onClick={handleToggleFullScreenPlayer}>
```

Comment:
Currently, this has weird behavior when you click on any of the metadata items in the bottom left. Specifically, it will try to open full screen player before doing some other operation. This includes the title (open to queue), artist (open to album page), album, and context menu.
Please either fix this behavior, or fix this behavior.
Also, personally, but this behavior doesn't make sense regardless, as there is no indicator that clicking anywhere else has any action.